app/helpers/goals_helper.rb

```diff
@@ -3,4 +3,15 @@ def time_left_to_complete(goal)
     time_of_expiration = goal.created_at + 7.days
     time_left_until_expiration = distance_of_time_in_words(Time.now, time_of_expiration)
   end
+
+  def urgency(goal)
+    time_elapsed = (Time.now - goal.created_at) / 1.day
```

Comment:
Good work calculating the time_elapsed here. GoalsHelper is a good choice for the responsibility of calculating the `time_elapsed` value. What model class might be a good choice for owning the responsibility of calculating this number?